postgresql/README.md

```diff
@@ -0,0 +1,70 @@
+---
+services: postgresql
+platforms: go
+author: gechris
+---
+
+# Azure PostgreSQL Samples
+
+This package demonstrates how to manage Azure VMs, their disks and container
+instances with the Go SDK.
+
+The child package "hybrid" demonstrates how to manage Azure VMs using Azure's
+Hybrid profile.
+
+## Contents
+
+* [How to run all samples](#run)
+* Management
+    * CreateServer - Create a PostgreSQL.
+    * UpdateServer - Updates a PostgreSQL server.
+    * DeleteServer - Deletes an existing PostgreSQL server.
+    * CreateOrUpdateFirewallRules - Creates or updates a firewall rule on the server.
+    * GetConfiguration - Get the configuration value that is set on the server.
+    * UpdateConfiguration - Updates the configuration.
+
+
+<a id="run"></a>
+## How to run all samples
+
+1. Get this package and all dependencies.
+
+  ```bash
+  export PROJECT=github.com/Azure-Samples/azure-sdk-for-go-samples/postgresql
+  go get -u $PROJECT
+  cd ${GOPATH}/src/${PROJECT}
+  dep ensure
+  ```
+1. Create an Azure service principal with the [Azure CLI][] command `az ad sp

```

Comment:
Seems we have a bad link here (Azure CLI)?

Reply:
Fixed